pkg/config/config.go

```diff
@@ -22,6 +22,9 @@ const (
 	kanikoDefaultImage = "gcr.io/kaniko-project/executor:v1.5.1"
 	kanikoImageEnvVar  = "KANIKO_CONTAINER_IMAGE"
 
+	remoteArtifactsDefaultImage = "busybox:latest"
```

Comment:
Let's use the full image reference?

Reply:
yes this will default to docker.io which can result in throttling

`quay.io/quay/busybox:latest`

Reply:
Indeed. Fixed on latest commit, using quay.io hosted image.